README.md

```diff
@@ -6,4 +6,20 @@ A declarative and deterministic framework for deploying and upgrading Cairo smar
 
 - Deploy new upgradeable contracts
 - Upgrade existing contracts
-- Declarative syntax for better developer experience
```

Comment:
Please, add again this line that surely was removed by accident :)

Reply:
the line is still there (line 9 under), it write as delete and add because of the end of line i think